Source/EventFlow/Extensions/EventFlowOptionsAggregatesExtensions.cs

```diff
@@ -62,7 +64,9 @@ public static class EventFlowOptionsAggregatesExtensions
             this EventFlowOptions eventFlowOptions,
             IEnumerable<Type> aggregateRootTypes)
         {
-            var invalidTypes = aggregateRootTypes
+            var aggregateRootTypeList = aggregateRootTypes.ToList();
```

Comment:
Interesting - thanks.

I found these too. 

http://weblogs.asp.net/lichen/linq-and-ienumerable-may-surprise-you
http://programmers.stackexchange.com/questions/194083/why-should-i-use-listt-over-ienumerablet